core/src/main/java/org/mobilitydata/gtfsvalidator/notice/NonAsciiOrNonPrintableCharNotice.java

```diff
@@ -18,7 +18,11 @@
 
 import com.google.common.collect.ImmutableMap;
 
-/** A row in the input file has a different number of values than specified by the CSV header. */
+/**
```

Comment:
This header doesn't match this notice. NonAsciiOrNonPrintableCharNotice should be a WARNING as discussed in https://github.com/MobilityData/gtfs-validator/pull/728